app/domain_models/record_sets_functions.rb

```diff
@@ -0,0 +1,67 @@
+class RecordSetsFunctions
+
+    attr_accessor :functions, :user_id, :contents, :sets, :reps, :weight, :record_sets_id
+    
+    def initialize(functions,account, contents, sets, reps, weight)
+        @functions = functions
+        @user_id = user_id
+        @contents = contents
+        @sets = sets
+        @reps = reps
+        @weight = weight
+    end
+
+    def set_functions
```

Comment:
Actually, in this class, there is not domain knowledge.
I think the domain knowledge is like, when you do some exercise, if the content, set, rep makes sense.

e.g. If users set the contents are running, clearly, sets and reps are not what the we expect to input.